assets/collection-filters-form.js

```diff
@@ -190,3 +181,16 @@ class PriceRange extends HTMLElement {
 }
 
 customElements.define('price-range', PriceRange);
+
+class FacetRemove extends HTMLElement {
+  constructor() {
+    super();
+    this.querySelector('a').addEventListener('click', (event) => {
```

Comment:
Good point - it might be a good idea to disable that link when there are no active filters for that category.  I'll make a note of it and fix it in another PR.